src/locales/translation-es.json

```diff
@@ -61,7 +61,11 @@
     "invalidLicenseAgreement": "Consentimiento no válido",
     "invalidURL": "URL no válida",
     "httpMissing": "falta \"http://\" o \"https://\")",
-    "generateHours": "¡Configura el horario de apertura!"
+    "generateHours": "¡Configura el horario de apertura!",
+    "links":  "Links y Social Media",
+    "linkUrl": "Link URL",
+    "linkTitle": "Link Título (opcional)",
+    "linkDescription": "Link Descripción (opcional)",
```

Comment:
Please fix wrong translation in this manner:
```
    "links": "Enlaces y redes sociales",
    "linkUrl": "URL del enlace",
    "linkTitle": "Título del enlace (opcional)",
    "linkDescription": "Descripción del enlace (opcional)",
```

Reply:
done on commit #[6fa6aee](https://github.com/kartevonmorgen/kartevonmorgen/pull/756/commits/6fa6aee2164af888c9f368b11e9fd369a40702fb)